dev-java/lombok-bin/lombok-bin-1.16.8.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+EAPI=6
+
+inherit eutils
+
+MY_PN="lombok"
```

Comment:
If you wanted, you could use use Bash magic `MY_PN=${PN%%-bin}`, instead. Not an issue, though.